### DISCUSSION TITLE
Anyway to increase workload?

### DISCUSSION THREAD

**Discussion by Cubey42** [volotat/SD-CN-Animation#25] [Ideas]
While I understand separating the flow for vram usage, my 4090 has a ton of overhead, is there any way to do do sets of frames now instead of just individual frames? or maybe like keep the controlnets loaded between frames or must they be reloaded?

**Comment by volotat** [comment]
Yeah. There is, but it requires some modification of the scripts. Right now, I'm working on the extension for web-ui and thinking about using another approach. Specifically, to unload SD weights run RAFT for N consecutive frames and then process it with SD. It should illuminate the need for extra steps with flow preparation, while keeping VRAM usage minimal. For people with big enough VRAM, I could do a special option that keeps both SD and RAFT in the memory simultaneously, although I highly doubt that this would dramatically improve performance. Regarding all other improvements, they are also possible, but as it is not the main concern for now, I’ll have to leave it for the future.

**Reply by Cubey42** [reply]
thanks for the reply! thanks for the continued work on this!
